.devcontainer/Dockerfile

```diff
@@ -3,5 +3,5 @@ FROM mcr.microsoft.com/vscode/devcontainers/python:0-${VARIANT}
 
 RUN apt-get update \
     && export DEBIAN_FRONTEND=noninteractive \
-    && apt-get install --no-install-recommends -y bc fontforge python-configparser \
+    && apt-get install --no-install-recommends -y bc fontforge \
```

Comment:
Was there a reason this was changed? I never noticed before but perhaps it's not needed to be installed separately in most cases?

Reply:
Yes, it causes the devcontainer to fail to build:
```
 > [2/2] RUN apt-get update     && export DEBIAN_FRONTEND=noninteractive     && apt-get install --no-install-recommends -y bc fontforge python-configparser     && fontforge --version 2>&1:
#5 0.494 Get:1 http://security.debian.org/debian-security bullseye-security InRelease [44.1 kB]
#5 0.591 Get:2 http://security.debian.org/debian-security bullseye-security/main amd64 Packages [97.5 kB]
#5 0.638 Get:3 https://dl.yarnpkg.com/debian stable InRelease [17.1 kB]
#5 0.922 Get:4 https://dl.yarnpkg.com/debian stable/main amd64 Packages [10.4 kB]
#5 0.977 Get:5 http://deb.debian.org/debian bullseye InRelease [116 kB]
#5 1.166 Get:6 https://dl.yarnpkg.com/debian stable/main all Packages [10.4 kB]
#5 1.529 Get:7 http://deb.debian.org/debian bullseye-updates InRelease [39.4 kB]
#5 1.537 Get:8 http://deb.debian.org/debian bullseye/main amd64 Packages [8180 kB]
#5 1.852 Get:9 http://deb.debian.org/debian bullseye/non-free amd64 Packages [93.8 kB]
#5 1.855 Get:10 http://deb.debian.org/debian bullseye/contrib amd64 Packages [50.5 kB]
#5 1.857 Get:11 http://deb.debian.org/debian bullseye-updates/main amd64 Packages [2592 B]
#5 2.698 Fetched 8662 kB in 2s (3780 kB/s)
#5 2.698 Reading package lists...
#5 3.167 Reading package lists...
#5 3.589 Building dependency tree...
#5 3.694 Reading state information...
#5 3.760 E: Unable to locate package python-configparser
```
Besides, it doesn't seem to be needed any more since the scripts still work without it. 🤷‍♂️

(Note: I created the original devcontainer, so I guess nobody else is using it? 🙄)

Reply:
Gotcha. thanks